Review.java

```diff
@@ -212,5 +212,6 @@ else if(sentiment<6)
 	}
 	return rating;
 }
+	

```

Comment:
@JordanTheDeveloper you should reput the code into review.java so that it shows that you made changes.